ChangeLog.d/error_const.txt

```diff
@@ -0,0 +1,6 @@
+Features
+   * New functions in the error module return constant strings for
+     high- and low-level error codes, complementing mbedtls_strerror
```

Comment:
Some ChangeLog entries added in this PR use `()` after function names. This entry doesn't.